src/common/SubmitCorrectionsButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+export default function SubmitCorrectionsButton({
+  id,
+}: {
+  id: string
+}): JSX.Element {
+  const email = 'support+corrections@caic.badw.de'
+  const subject = encodeURIComponent(`eBL Correction to ${id}`)
+  const body = encodeURIComponent(
+    `To the ${id}, I have the following correction:\n\n[comment]`
+  )
+  return (
+    <Button
```

Comment:
Maybe add `title="Suggest correction"` which will appear as tooltip when hovering the mouse over the mail icon.